CONTRIBUTING.rst

```diff
@@ -0,0 +1,93 @@
+============
+Contributing
+============
+
+Contributions are welcome and appreciated!
+Every little bit helps, and a credit will always be given.
+
+.. _issues : https://github.com/nexB/scancode-workbench/issues
+__ issues_
+
+If you are new to ScanCode Workbench and want to find easy tickets to work on,
+check `easy issues <https://github.com/nexB/scancode-workbench/labels/easy>`_
+
+When contributing to ScanCode Workbench (such as code, bugs, documentation, etc.) you
+agree to the Developer `Certificate of Origin <http://developercertificate.org/>`_
+and the ScanCode license (see the `NOTICE <https://github.com/nexB/scancode-workbench/blob/develop/NOTICE>`_ file).
+The same approach is used by Linux Kernel developers and several other projects.
+
+For commits, it is best to simply add a line like this to your commit message,
+with your name and email::
+
+    Signed-off-by: Jane Doe <developer@example.com>
+
+Please try to write a good commit message, see `good commit message wiki
+<https://aboutcode.readthedocs.io/en/latest/contributing/writing_good_commit_messages.html>`_ for
+details. In particular use the imperative for your commit subject: think that
+you are giving an order to the codebase to update itself.
+
+
+Feature requests and feedback
+=============================
+
+To send feedback or ask a question, `file an issue <issues_>`_
+
+If you are proposing a feature:
+
+* Explain how it would work.
+* Keep the scope as simple as possible to make it easier to implement.
+* Remember that your contributions are welcomed to implement this feature!
+
+
+Chat with other developers
+==========================
+
+For other questions, discussions, and chats, we have:
+
+- an official Gitter channel at https://gitter.im/aboutcode-org/discuss
```

Comment:
Let's update this with https://matrix.to/#/#aboutcode-org_discuss:gitter.im and https://matrix.to/#/#aboutcode-org_scancode-workbench:gitter.im

Reply:
thanks for pointing this out